src/components/ButtonsToolbar/test.spec.js

```diff
@@ -141,3 +145,32 @@ it('Non-last step of non-submittable form without overview, button IS present',
   expect(buttons[1].textContent).toEqual('Save step');
   expect(buttons[2].textContent).toEqual('Next step');
 });
+
+it('Suspending form not allowed, button is NOT present', () => {
+  const mockFunction = jest.fn();
+
+  act(() => {
+    render(
+      <ButtonsToolbar
+        literals={LITERALS}
+        canSubmitStep={true}
+        canSubmitForm={SUBMISSION_ALLOWED.yes}
+        canSuspendForm={false}
+        isAuthenticated={false}
+        isLastStep={true}
+        isCheckingLogic={false}
+        loginRequired={false}
+        onNavigatePrevPage={mockFunction}
+        onFormSave={mockFunction}
+        onLogout={mockFunction}
+      />,
+      container
+    );
+  });
+
+  const buttons = container.getElementsByClassName('openforms-toolbar__list-item');
+
+  expect(buttons.length).toEqual(2);
+  expect(buttons[0].textContent).toEqual('Previous step');
+  expect(buttons[1].textContent).toEqual('Next step');
```

Comment:
please make the assertion that the 'Save step' literal is not present, as that is what the test/prop is about. If we add or tweak another unrelated button, this test will break while the `canSuspendForm` behaviour has not changed.

I'm really expecting something like `expect(await screen.queryByRole('button', {name: 'Save step'}).toBeNull()`

Reply:
The nicest way to do this would be to introduce something like testing library here, but that would either differentiate from the other tests in the same file (like testing that the submit button is hidden).  I'd rather have a consistent testing approach for all the tests for a component. Do we want to refactor the other tests as well?

Reply:
I thought testing-library is already set up?

```
➜  open-forms-sdk git:(main) yarn list --pattern=testing-library
yarn list v1.22.19
├─ @storybook/jest@0.0.10
│  └─ @testing-library/jest-dom@5.16.5
├─ @storybook/testing-library@0.0.13
│  └─ @testing-library/user-event@13.5.0
├─ @testing-library/dom@8.16.0
├─ @testing-library/jest-dom@5.11.10
├─ @testing-library/react@11.2.6
│  └─ @testing-library/dom@7.30.3
├─ @testing-library/user-event@12.8.3
├─ @types/testing-library__jest-dom@5.9.5
└─ eslint-plugin-testing-library@5.4.0
```

Reply:
Yes testing library is already available so can easily be added and I agree that making sure something doesn't exist is more accurate (and descriptive).

I find different patterns in similar tests confusing but as discussed: I will create an issue for this.